module-apps/apps-common/ApplicationCommon.cpp

```diff
@@ -1058,4 +1060,37 @@ namespace app
         getCurrentWindow()->updateBatteryStatus();
         getCurrentWindow()->updateSim();
     }
+
+    void ApplicationCommon::refreshTimeOnPhoneLockWindow()
+    {
+        lockScreenRefreshCounter.currentValue++;
+        if (lockScreenRefreshCounter.currentValue >= lockScreenRefreshCounter.referenceValue) {
+            lockScreenRefreshCounter.currentValue = 0;
+            LOG_DEBUG("Requesting deep refresh on phone lock screen");
+            refreshWindow(gui::RefreshModes::GUI_REFRESH_DEEP);
+        }
+        else {
+            LOG_DEBUG("Requesting fast refresh on phone lock screen");
+            refreshWindow(gui::RefreshModes::GUI_REFRESH_FAST);
+        }
+    }
+
+    void ApplicationCommon::initializeDeepRefreshCounter()
+    {
+        constexpr auto fallbackRefreshRate = 15;
```

Comment:
why fallback is different than the DB value?
consider using `std::chrono`'s literals

Reply:
My mistake, fixed now.
As for literals, I don't think using them here is required. I'm only counting calls of this function to count the time, not using some "real" timer ;)